<launch>
    <node name="closedloopfbpoint2mag"
        pkg ="plane_camera_magnet"
        type = "closedloopfbpoint2mag"
        output ="screen"
        launch-prefix = "gdb -ex run --args">
        <param name="param_filename" value="/home/denise/catkin_ws/src/plane_camera_magnet/calib/closedloopfbpoint.yml"/>
    </node>
</launch>